client/src/components/NewPastaForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+const NewPastaForm = ({ postPasta }) => {
+    const [newPasta, setNewPasta] = useState({
+    name: "",
+    description: "",
+  });
+
+  const handleInputChange = event => {
```

Comment:
nit-picking consistency - this function is without parameter parenthesis, but the next function includes them